Publish Ros Maps - Redirects rosinstall urls

BarceloChristian opened the pull request:
- Add parameter to add a transform tolerance to the TF stamp.
- Add parameter to publish the occupancy grid as Ros standard
- Change rosinstall sources

BarceloChristian committed:
Add transform tolerance parameter

BarceloChristian committed:
Add publish ros map parameter

bochen87 committed:
Fix broken build for melodic. Use official ros docker image. (#1484)

Essentially revert #1021 as image seems to be fixed

Signed-off-by: Bo Chen <bo@enway.ai>

bochen87 committed:
Change Google CLA to DCO in CONTRIBUTING.md (#1482)

eborghi10 commented on cartographer_ros/cartographer_ros/msg_conversion.cc:
```diff
@@ -404,10 +404,17 @@ std::unique_ptr<nav_msgs::OccupancyGrid> CreateOccupancyGridMsg(
+      if (observed != 0) {
+        const auto cell = ::cartographer::common::RoundToInt((1. - color / 255.) * 100.);
+        if (publish_ros_map) {
+          if (cell < 49) value = 0;
```
What' the idea behind this?

BarceloChristian replied:
The idea is to set the values into the trinary interpretation, so MoveBase can plan through this map, and the exploratory packages can find frontiers.

eborghi10 replied:
Can you add this explanation into the code? What I don't understand is why those values and not others?

wohe committed:
Fix build status on front page. (#1483)

* Fix build status on front page.

Changes "googlecartographer" to "cartographer-project"
for references to CI and GitHub.

Following cartographer-project/cartographer#1693.

Signed-off-by: Wolfgang Hess <whess@lyft.com>

* Remove Indigo and Lunar Dockerfiles.

BarceloChristian committed:
Fix Travis

wohe committed:
Build Abseil dependency in CI. (#1485)

cartographer-project/cartographer#1711 removed
building Abseil from the Cartographer library.
We follow here the same approach as in
cartographer CI.

An alternative approach which is now possible is
adding an Abseil package for catkin and depending
on that.

Signed-off-by: Wolfgang Hess <whess@lyft.com>

BarceloChristian committed:
Change rosinstall sources
